### PR TITLE
Fix nightlies by bumping tasty-hedgehog

### PR DESCRIPTION
**Pull request by martijnbastiaan**

### CODE REVIEW

File: clash-prelude/tests/Test/Tasty/Hedgehog/Extra.hs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module Test.Tasty.Hedgehog.Extra (testPropertyXXX) where
```
Comment (alex-mckenna) [inline clash-prelude/tests/Test/Tasty/Hedgehog/Extra.hs:1]: I know we've been historically bad about this, but can you put in a haddock module header with the copyright?

**Review by alex-mckenna — APPROVED** [review]

File: clash-prelude/tests/Test/Tasty/Hedgehog/Extra.hs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{-|
+Convenience functions \"missing\" in "Test.Tasty.Hedgehog"
+
+Copyright  :  (C) 2022, QBayLogic B.V.
+License    :  BSD2 (see the file LICENSE)
+Maintainer :  QBayLogic B.V. <devops@qbaylogic.com>
```
Comment (alex-mckenna) [inline clash-prelude/tests/Test/Tasty/Hedgehog/Extra.hs:6]: Doesn't the module header have to be before the description?

Comment (martijnbastiaan) [reply clash-prelude/tests/Test/Tasty/Hedgehog/Extra.hs]: Jup

**Comment by alex-mckenna** [comment]
:fork_and_knife: :hedgehog:

**Pull request merged**